src/rdkafka_conf.c

```diff
@@ -779,6 +779,25 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 		} },
         { _RK_GLOBAL|_RK_PRODUCER, "compression.type", _RK_C_ALIAS,
           .sdef = "compression.codec" },
+	{ _RK_GLOBAL|_RK_PRODUCER, "compression.level", _RK_C_INT,
```

Comment:
librdkafka now supports default topic configuration to be set on the global object (it instanties a default topic config object itself), we don't need to add this to the global config but only to the topic one.

Reply:
OK, done in 81b010d
What about `compression.codec`. Shouldn't it be the same?

Reply:
Oups! Forgot to actually remove the field from config structure. Done in f7584f8